app/views/selfupdate_beta.scala.txt

```diff
@@ -140,10 +140,6 @@ if [[ -z $(cat ${sdkman_config_file} | grep 'sdkman_debug_mode') ]]; then
 	echo "sdkman_debug_mode=true" >> "$sdkman_config_file"
 fi
 
-if [[ -z $(cat ${sdkman_config_file} | grep 'sdkman_colour_enable') ]]; then
```

Comment:
Not sure why we are deleting the `sdkman_colour_enable` config. Is this intentional?

Reply:
Probably wasn't a good idea to make the change with two kids running around me screaming 🤣.